tests/http/scorecard.py

```diff
@@ -45,17 +45,19 @@ class ScoreCardError(Exception):
 class ScoreCard:
 
     def __init__(self, env: Env,
-                 httpd: Optional[Httpd],
-                 nghttpx: Optional[Nghttpx],
-                 caddy: Optional[Caddy],
+                 protocol: str,
+                 server_descr: str,
+                 server_port: Union[str, int],
```

Comment:
ports are pretty consistently int in the code. I think this can simply be int here instead of a union.

Reply:
There was one case where it was parsed as str and I changed that and removed the Union again.